crates/storage/src/collections/binary_heap/mod.rs

```diff
@@ -50,6 +51,14 @@ where
 {
     /// The individual elements of the heap.
     elements: ChildrenVec<T>,
+    /// The offset key for the N cells.
+    ///
+    /// If the lazy chunk has been initialized during contract initialization
+    /// the key will be `None` since there won't be a storage region associated
+    /// to the lazy chunk which prevents it from lazily loading elements. This,
+    /// however, is only checked at contract runtime. We might incorporate
+    /// compile-time checks for this particular use case later on.
+    key: Option<Key>,
```

Comment:
What exactly has this to do with the PR? :sweat_smile:

Reply:
Argh, it was a survivor from a different approach. Removed it now.